### PR TITLE
tutorial code updates

### PR DESCRIPTION
**Pull request by dburles**

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -58,12 +58,14 @@ var App = React.createClass({
...
-      this.data.tasks.map(function (task) {
-        return <li key={task._id}>{task.content}</li>;
-      })
-    }</ul>
+    return (
+      <ul>
+        {/* Access the data from trackMeteorData() on this.data */}
+        {this.data.tasks.map(function (task) {
+          return <li key={task._id}>{task.content}</li>;
+        })}
```
Comment (stubailo) [inline docs/tutorial.md]: This is just bracket placement - is there a standard/best practice for this?

File: docs/tutorial.md [hunk 2]
```diff
@@ -105,27 +109,31 @@ var NewTaskForm = React.createClass({
...
   }
 })
 
 var App = React.createClass({
   render() {
-    return <div>
-      <List />
-      <NewTaskForm />
-    </div>
+    return (
```
Comment (stubailo) [inline docs/tutorial.md]: What's the benefit of parentheses here?

Comment (stubailo) [inline docs/tutorial.md]: OK, I see that this matches the React tutorial itself.

Comment (dburles) [inline docs/tutorial.md]: Yep and the style used in their docs

File: docs/tutorial.md (hunk 1 above)
Comment (dburles) [inline docs/tutorial.md]: As far as I have seen this seems to be the more commonly used style, as blocks of javascript within jsx are "just syntactic sugar for function calls and object construction." https://facebook.github.io/react/tips/if-else-in-JSX.html also see https://facebook.github.io/react/docs/jsx-in-depth.html#comments

**Comment by stubailo** [comment]
LGTM, thanks for doing the research. Going to update the example app when I get the chance.

**Comment by dburles** [comment]
Great, thanks

**Pull request merged**